src/Workspaces/Core/Portable/CodeGeneration/CodeGenerationSymbolFactory.cs

```diff
@@ -295,14 +295,15 @@ public static ITypeParameterSymbol CreateTypeParameterSymbol(string name, int or
             ImmutableArray<AttributeData> attributes,
             VarianceKind varianceKind, string name,
             ImmutableArray<ITypeSymbol> constraintTypes,
+            NullableAnnotation nullableAnnotation = NullableAnnotation.None,
```

Comment:
@ryzngard I'm confused by the question: this is the helper method that _creates_ an ITypeParameterSymbol out of thin air, so it needs to come along.

Reply:
I meant to move this down to the `CreateArrayTypeSymbol`, but I realize it doesn't matter there either because array nullability != array type nullability.